Formula/pandemics.rb

```diff
@@ -0,0 +1,28 @@
+require "language/node"
+
+class Pandemics < Formula
+  desc "Converts your markdown document in a simplified framework"
+  homepage "https://pandemics.gitlab.io"
+  url "https://registry.npmjs.org/pandemics/-/pandemics-0.11.11.tgz"
+  sha256 "e9e54497f5b32c15cb5635d780566d855d086b82c352be154d5fb01a18f8ff7e"
+  license "BSD-3-Clause"
+
+  depends_on "librsvg"
+  depends_on "node"
+  depends_on "pandoc"
+  depends_on "pandoc-crossref"
+
+  def install
+    ENV["PANDEMICS_DEPS"]="0"
+    # npm ignores config and ENV when in global mode so:
+    # - install without running the package install script
+    system "npm", "install", "--ignore-scripts", *Language::Node.std_npm_install_args(libexec)
+    # - call install script manually to ensure ENV is respected
+    system "npm", "run", "--prefix", "#{libexec}/lib/node_modules/pandemics", "install"
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_equal version, shell_output("#{libexec}/bin/pandemics --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thank you for the feedback.
I've added a test to check that the app main (and only, so far) functionality, that is to compile markdown documents, is working properly. This test also implicitly checks that dependencies are found.